api/tests/AppBundle/Entity/Repository/DocumentRepositoryTest.php

```diff
@@ -212,7 +212,7 @@ public function supportsWhenDocumentsForMoreThanOneReportAreQueued()
         $this->assertDataMatchesEntity($documents, $reportPdfDoc2, $client2, $reportSubmission2, $report2);
         $this->assertDataMatchesEntity($documents, $supportingDoc2, $client2, $reportSubmission2, $report2);
 
-        foreach([$reportPdfDoc, $supportingDoc,$reportPdfDoc, $supportingDoc] as $doc) {
+        foreach([$reportPdfDoc, $supportingDoc, $reportPdfDoc, $supportingDoc] as $doc) {
```

Comment:
Just noticed - are the second `$reportPdfDoc` and `$supportingDoc` supposed to be `$reportPdfDoc2` and `$supportingDoc2`?

Reply:
Not really sure. Wasn't one of my changes tests. Just thought I'd sort out some other missing spaces.. That test does look well dodge but as it's running and not part of what i was doing I don't want to have to rerun the workflow for it :)